lending/loan_management/report/loan_statement_of_account/loan_statement_of_account.html

```
@@ -0,0 +1,157 @@
+				<td style="border:1.5px solid black;">{%= __("Transaction Type").toLocaleUpperCase() %}</td>
+				{% if (filters.group_by !== "Grouped") { %}
+					<td style="border:1.5px solid black;">{%= __("Transaction").toLocaleUpperCase() %}</td>
+				{% } %}
+				<td style="border:1.5px solid black;">{%= __("Loan").toLocaleUpperCase() %}</td>
+				<td style="border:1.5px solid black; width:9em;">{%= __("Debit").toLocaleUpperCase() %}</td>
+				<td style="border:1.5px solid black; width:9em;">{%= __("Credit").toLocaleUpperCase() %}</td>
+				<td style="border:1.5px solid black; width:10em;">{%= __("Balance").toLocaleUpperCase() %}</td>
+			</tr>
+		</thead>
+		<tbody>
+			{% for(var i=0, l=data.length; i<l; i++) { %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)